tests/test_reverse_one_to_one_relation_crudl_operations_are_supported.py

```
@@ -0,0 +1,176 @@
+    amz_author_profile = models.AmazonAuthorProfile.objects.create(
+        description="Some description",
+    )
+
+    response = client.post(
+        "/api/authors",
+        content_type="application/json",
+        data={
+            "name": "Some author",
+            "birth_date": "1990-01-01",
+            "books": [book.id],
+            # TODO(phuongfi91): The field should not be required
```

Ruff (0.8.2)

Missing issue link on the line following this TODO

(TD003)